### PR TITLE
Implemented HTTP harvest plugin

### PR DESCRIPTION
**Pull request by SidSatbhai**

### CODE REVIEW

File: metis-core/metis-core-common/src/main/java/eu/europeana/metis/core/workflow/plugins/HTTPHarvestPlugin.java [hunk 1]
```diff
@@ -1,40 +1,63 @@
...
+
+		Map<InputDataType, List<String>> dataEntries = new EnumMap<>(InputDataType.class);
+		List<String> urls = new ArrayList<>();
+		urls.add(httpUrl);
+		dataEntries.put(InputDataType.REPOSITORY_URLS, urls);
+		dpsTask.setInputData(dataEntries);
+
+		Map<String, String> parameters = new HashMap<>();
+		parameters.put("PROVIDER_ID", ecloudProvider);
+		parameters.put("OUTPUT_DATA_SETS", String.format(CommonStringValues.S_DATA_PROVIDERS_S_DATA_SETS_S_TEMPLATE));
```
Comment (stzanakis) [inline metis-core/metis-core-common/src/main/java/eu/europeana/metis/core/workflow/plugins/HTTPHarvestPlugin.java]: String.format is missing the 3 required parameters.

File: metis-core/metis-core-common/src/main/java/eu/europeana/metis/core/workflow/plugins/HTTPHarvestPlugin.java [hunk 2]
```diff
@@ -1,40 +1,63 @@
...
+		Map<InputDataType, List<String>> dataEntries = new EnumMap<>(InputDataType.class);
+		List<String> urls = new ArrayList<>();
+		urls.add(httpUrl);
+		dataEntries.put(InputDataType.REPOSITORY_URLS, urls);
+		dpsTask.setInputData(dataEntries);
+
+		Map<String, String> parameters = new HashMap<>();
+		parameters.put("PROVIDER_ID", ecloudProvider);
+		parameters.put("OUTPUT_DATA_SETS", String.format(CommonStringValues.S_DATA_PROVIDERS_S_DATA_SETS_S_TEMPLATE));		
+		dpsTask.setParameters(parameters);
```
Comment (stzanakis) [inline metis-core/metis-core-common/src/main/java/eu/europeana/metis/core/workflow/plugins/HTTPHarvestPlugin.java]: Can you also add the parameter NEW_REPRESENTATION_NAME check OaipmhHarvestPlugin to do the same.

File: metis-core/metis-core-common/src/main/java/eu/europeana/metis/core/workflow/plugins/HTTPHarvestPlugin.java (hunk 1 above)
Comment (SidSatbhai) [reply metis-core/metis-core-common/src/main/java/eu/europeana/metis/core/workflow/plugins/HTTPHarvestPlugin.java]: Added.

File: metis-core/metis-core-common/src/main/java/eu/europeana/metis/core/workflow/plugins/HTTPHarvestPlugin.java (hunk 2 above)
Comment (SidSatbhai) [reply metis-core/metis-core-common/src/main/java/eu/europeana/metis/core/workflow/plugins/HTTPHarvestPlugin.java]: Added.

**Pull request merged**